TypeCobol.Test/Parser/Incremental/BasicEdits/AddEmptyLineAtEndThenRemove/Typedef2.tcbl

```diff
@@ -0,0 +1,18 @@
+﻿IDENTIFICATION DIVISION.
```

Comment:
My guess is that the latest merge from develop added the `MyPgm` test but did not remove this one because it was added by another commit previously. Anyway I removed it manually.